website/content/v1.11/talos-guides/configuration/logging.md

```diff
@@ -239,6 +239,9 @@ config:
       Daemon        Off
       Log_Level     warn
       Parsers_File  custom_parsers.conf
+      HTTP_Server   On
```

Comment:
is it safe to expose this?

Reply:
It exposes metrics about fluentbit and not actual logs so I assume it would be safe and necessary if someone is scraping it with prometheus